simulator-ui/src/main/java/org/citrusframework/simulator/ui/filter/SpaWebFilter.java

```diff
@@ -0,0 +1,37 @@
+package org.citrusframework.simulator.ui.filter;
+
+import jakarta.annotation.Nullable;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.citrusframework.simulator.http.SimulatorRestConfigurationProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.web.util.matcher.RequestMatcher;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+public class SpaWebFilter extends OncePerRequestFilter {
```

Comment:
I also want to write a unit-test for this class:
- [ ] write a Unit-Test for the `SpaWebFilter`